apps/meteor/app/federation-v2/server/infrastructure/rocket-chat/adapters/Settings.ts

```diff
@@ -30,9 +30,15 @@ export class RocketChatSettingsAdapter {
 	}
 
 	public getBridgePort(): number {
-		const [, , port] = this.getBridgeUrl().split(':');
+		const defaultPort = 3334;
```

Comment:
I don't think we should hardcode a default port here. In my opinion, it would be better to find the root problem of having the port undefined.